src/main/java/com/spring/taskmanger/controller/UserController.java

```diff
@@ -24,36 +28,37 @@
 	@Autowired
 	UserService userService;
 
-	@GetMapping("/users")
-	public List<User> getAllUsers() {
-
-		return userService.getAllUsers();
-	}
+	@Autowired
+	TokenRepository tokenRepository;
 
-	@GetMapping("/users/{id}")
+	@GetMapping("/users")
 	// ResponseEntity represents the whole HTTP response: status code, headers, and
 	// body, Used in RestTemplate as well @Controller methods.
-	public ResponseEntity<User> getUser(@PathVariable Long id) {
+	public ResponseEntity<User> getUser() {
+		return userService.getUser();
+	}
+
+	@PutMapping("/users")
+	public ResponseEntity<User> UpdateUser(@Valid @RequestBody User user) {
+		return userService.updateUser(user);
 
-		return userService.getUser(id);
+	}
+
+	/*
+	@GetMapping("/users")
+	public List<User> getAllUsers() {
+		return userService.getAllUsers();
 	}
 
 	@PostMapping("/users")
```

Comment:
Please clean your code, remove any unneeded piece of code

Reply:
OK